training/02_Getting_Started/05_config_env_azure.adoc

```diff
@@ -0,0 +1,75 @@
+=== Workstation setup for Azure
+
+==== Prerequisites
+
+This is a configuration guide of the packages and repos you need to install the tools, but a previoulsy existing Azure account is needed.
+
+==== Steps
+
+If you want to deploy on azure you will need the Azure client.
+
+https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest[Source documentation]
+
+.in a nutshell (tested on fedora 28) - Azure cli (system-wide)
+----
+
+# Install the azure-cli system-wide
+sudo -i
+rpm --import https://packages.microsoft.com/keys/microsoft.asc
+cat >> /etc/yum.repos.d/azure-cli.repo <<EOF
+[azure-cli]
+name=Azure CLI
+baseurl=https://packages.microsoft.com/yumrepos/azure-cli
+enabled=1
+gpgcheck=1
+gpgkey=https://packages.microsoft.com/keys/microsoft.asc
+EOF
+
+yum check-update
+yum install -y azure-cli
+----
+
+We recommend you install the ansible module in a virtualenv.
+
+.in a nutshell (tested on fedora 28) - Azure ansible module (use virtualenv)
+----
+# /!\ careful this will update ansible as well
+# Use a virtualenv for those:
+pip install --upgrade pip
+pip install --upgrade --force ansible[azure]
+----
+
+NOTE: `--force` is used here, because of a known link:https://github.com/ansible/ansible/issues/38894[issue].
+
+==== Service principal
+
+It's better to use a service principal instead of your main credentials. Refer to the https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli?view=azure-cli-latest[official documentation].
+
+.in a nutshell
+----
+az login
+az ad sp create-for-rbac
+az login --service-principal -u <user> -p <password-or-cert> --tenant <tenant>
+----
+
+.env_secret_vars.yml
+----
+azure_service_principal: "service principal client id"
+azure_password: "service principal password or cert"
+azure_tenant: "tenant ID"
+azure_region: "Azure location, ex: EuropeWest"
+azure_subscription_id: "Subscription id"
+----
+
+==== Virtualenv
+
+If you want to use virtualenv, you can try & adapt this:
+
+----
+cd ansible
+mkdir ~/virtualenv-aad
+virtualenv ~/virtualenv-aad -p python2.7
+. ~/virtualenv-aad/bin/activate
+export CC=gcc-5
+pip install -r requirements.txt
```

Comment:
The contents for requirements.txt can be found in the repo under tools/virtualenvs/azure-ansible-latest.txt to save you the trouble, and python 3 is recommended so "python3 -m venv <directory>"